e2e/testcases/helm_sync_test.go

```diff
@@ -105,14 +103,16 @@ func TestHelmNamespaceRepo(t *testing.T) {
 		ReleaseName: "test",
 		SecretRef:   v1beta1.SecretReference{Name: "foo"},
```

Comment:
Consider adding a `Cleanup` block to delete the `foo` Secret as part of the test cleanup

Reply:
Probably not needed for RepoSync test, since the repo sync namespace will be deleted and this secret is under that namespace.  But for RootSync test, since the secret is under config-management-system namespace,  It is better that clean up the secret after test finish. Will add this with other helm e2e test changes.